go/api/config.go

```diff
@@ -226,32 +282,34 @@ func (config *RedisClusterClientConfiguration) WithAddress(address NodeAddress)
 	return config
 }
 
-// WithUseTLS configures the TLS settings for this configuration. Set to true if communication with
-// the cluster should use Transport Level Security. This setting should match the TLS configuration
-// of the server/cluster, otherwise the connection attempt will fail.
+// WithUseTLS configures the TLS settings for this configuration. Set to true if communication with the cluster should use
+// Transport Level Security. This setting should match the TLS configuration of the server/cluster, otherwise the connection
+// attempt will fail.
 func (config *RedisClusterClientConfiguration) WithUseTLS(useTLS bool) *RedisClusterClientConfiguration {
 	config.useTLS = useTLS
 	return config
 }
 
-// WithCredentials sets the credentials for the authentication process. If none are set, the client
-// will not authenticate itself with the server.
-func (config *RedisClusterClientConfiguration) WithCredentials(credentials *RedisCredentials) *RedisClusterClientConfiguration {
+// WithCredentials sets the credentials for the authentication process. If none are set, the client will not authenticate
+// itself with the server.
+func (config *RedisClusterClientConfiguration) WithCredentials(
+	credentials *RedisCredentials,
+) *RedisClusterClientConfiguration {
 	config.credentials = credentials
 	return config
 }
 
-// WithReadFrom sets the client's [ReadFrom] strategy. If not set, [PRIMARY] will be used.
+// WithReadFrom sets the client's [ReadFrom] strategy. If not set, [Primary] will be used.
```

Comment:
Probably you can omit function names in all code docs

Reply:
That's what I thought as well, but the [Go docs](https://tip.golang.org/doc/comment) page specifies that they should be included:
"As with packages (above) and funcs (below), doc comments for types start with complete sentences naming the declared symbol."
If I remove the function name, my IDE gives me a warning that the doc comment should start with the function name.

Reply:
Good to know, thanks. That is a Go specific thing.